src/documents/components/FileDetail/SelectElement.js

```diff
@@ -24,30 +24,31 @@ const SelectElement = ({
         setFieldValue(name, id);
       }}
     >
-      <Radio
-        className="selectionRadio"
-        name={name}
-        id={id}
-        value={id}
-        checked={selected}
-        onChange={onChange}
-        label={() => (
-          <Icon
-            name={fileTypeDetail[id].icon}
-            size="large"
-            bordered
-            circular
-            inverted
-            color={selected ? 'blue' : 'black'}
-          />
-        )}
-      />
       <div>
-        <p>
+        <Radio
+          name={name}
+          id={id}
+          value={id}
+          checked={selected}
+          onChange={onChange}
+          label={() => (
+            <Icon
+              name={fileTypeDetail[id].icon}
+              size="large"
+              bordered
+              circular
+              inverted
+              color={selected ? 'blue' : 'black'}
+            />
+          )}
+        />
+      </div>
+      <div>
+        <p className="mb-5">
           <b>{fileTypeDetail[id].title}</b>
         </p>
-        <p>
-          <small>{fileTypeDetail[id].description}</small>
+        <p className="text-wrap-75 line-height-small text-grey">
```

Comment:
looking at this more I think we can remove the `line-height-small`, I got the leading a bit tight and it makes it too inconsistent with the type at the top of the page